templates/_components/_image.twig

```diff
@@ -3,13 +3,154 @@
 {#
   Image
   
-  Generate image transforms and render images
-  
-  @param   asset        {AssetFileModel}    Image asset to be rendered
-  @param   preset       {string}            Image usage preset
-  @param   class        {string}            Class names to apply to the img tag
-  @param   alt          {string}            Value of the alt attribute for the img tag
-  @param   lazyload     {bool}              Lazyload image or not
+  Generate, transform, manipulate, optimize and render images. 
+
+	What it can do:
+	---------------
+	1. Auto-generates transforms and responsive images via `sizes` and `srcset` attributes 
+	2. Auto-crops images if needed/specified
+	3. Auto-generates art-directed responsive images using the `<picture>` and `<source>` tags
+	4. Auto-generates WebP versions (if supported)
+	5. Lazyload images using lazysizes
+	6. Allows images to be specified as CSS class background images
+	7. Allows use of different source images for different breakpoints
+	8. And finally, renders the image
+
+	How to use:
+  -----------
+	Include the `_image` component with the following parameters where you want to display the image.
+
+	Configuration parameters:
+	@param   asset                {AssetFileModel}          [Required] Image asset to be rendered
+	@param   breakpoint_assets    {array|AssetFileModel}    Array of image assets to be rendered for different breakpoints
```

Comment:
> Array of image assets to be rendered for different breakpoints

Just _Image assets to be rendered for different breakpoints_ is sufficient. Data type is already specified in the third column.

Reply:
Done.